src/app/components/modal-create-project/modal-create-project.component.ts

```diff
@@ -14,33 +14,55 @@ import {Subject} from 'rxjs';
 })
 export class ModalCreateProjectComponent implements OnInit {
 
-  public readonly onClose: Subject<any> = new Subject<any>();
-
   projCreateForm: FormGroup;
   rapidFolder: RemoteFile;
   submitting: boolean;
   errorMessage = '';
+  currentPath: string;
+  fileSuffix: string = '';
```

Comment:
should this be renamed to "fileName" or "fileBaseName" as the suffix is always "hazmapper".